CHANGELOG.md

```diff
@@ -1,5 +1,96 @@
 # Change Log
 
+## v0.10.0 2021-02-01
+
+[full changelog](https://github.com/executablebooks/jupyter-book/compare/v0.9.1...d3c78097edda4fefb672e32344c3806c9cdc7a72) | [GitHub contributors page for this release](https://github.com/executablebooks/jupyter-book/graphs/contributors?from=2020-12-22&to=2021-01-30&type=c)
+
+This update focuses on new syntax features for MyST markdown, as well as a new configuration to enable MyST extensions. See below for more details.
+
+### New
+
+**MyST Parser version 0.13.x**
+: The MyST-NB and MyST-parser have both been upgraded.
+  This comes with support for new syntax and a new configuration mechanism (see below for some examples).
+
+  See [the myst-parser changelog](https://myst-parser.readthedocs.io/en/latest/develop/_changelog.html#id4) for more information about the syntax additions.
+
+**New `myst` extension configuration**
+: The `myst_extended_syntax` configuration is **now deprecated**, in lieu of a more flexible extension mechanism.
+  You may now enable individual `myst` extensions by adding them to the following section of your `_config.yml` file:
+  ```yaml
+  parse:
+    myst_enable_extensions:
+      - <list-of-extensions>
+  ```
+  See [](content-blocks:myst-extensions) for more information.
```

Comment:
I mentioned to @mmcky last release (https://github.com/executablebooks/jupyter-book/pull/1129#discussion_r538992853), you should perhaps try to avoid reference links in the changelog, since people are likely to also read this document directly  on Github (i.e. from https://github.com/executablebooks/jupyter-book/blob/master/CHANGELOG.md), where such links will be unusable.
e.g. instead something like this might be better:

`[the MyST syntax extension section here](docs/content/content-blocks.md)`

Note, there is actually the option in myst-parser to activate header anchors: https://myst-parser.readthedocs.io/en/latest/using/syntax-optional.html#auto-generated-header-anchors, but this is not "turned on" (or documented) in jupyter book